ui/src/utils/labels.js

```diff
@@ -1,6 +1,6 @@
 export default {
   RUNNING: 'info',
-  PENDING: 'default',
+  PENDING: 'warning',
```

Comment:
We use `warning` for exception already. Also, the filter card color should change as well:

![Screen Shot 2019-11-06 at 1 37 10 PM](https://user-images.githubusercontent.com/3766511/68327045-9ac84080-009a-11ea-9786-653ffd8590e7.png).

Reply:
Yeah, but all others are also in use already. What do you suggest in that case?

Reply:
That's a good question. We could probably just use a class to override the color.

`<StatusLabel className={...} />`

Reply:
Hm.. But Isn't `StatusLabel `is used in various components?